Source/Icebreaker/Helpers/IcebreakerBotDataProvider.cs

```diff
@@ -326,6 +348,14 @@ private async Task InitializeAsync()
             usersCollectionDefinition.PartitionKey.Paths.Add("/id");
             this.usersCollection = await this.documentClient.CreateDocumentCollectionIfNotExistsAsync(this.database.SelfLink, usersCollectionDefinition, useSharedOffer ? null : requestOptions);
 
+            // Get a reference to the Feedback collection, creating it if needed
+            var feedbackCollectionDefinition = new DocumentCollection
+            {
+                Id = feedbackCollectionName
+            };
+            feedbackCollectionDefinition.PartitionKey.Paths.Add("/id");
```

Comment:
NIT: it might be useful to call out that this is just randomly partitioning the data since there isn't an `id` field on the `UserFeedback` document

Reply:
Would it be better to partition by teamId?